### PR TITLE
Add supported source databases to DMS docs

### PR DESCRIPTION
**Pull request by nickvigilante**
Add supported source databases to DMS docs

### CODE REVIEW

**Review by otan — APPROVED** [review]

File: v22.1/aws-dms.md [hunk 1]
```diff
@@ -25,6 +25,18 @@ Ensure the following items are completed prior to starting this tutorial:
...
 - Ensure you have a secure, publicly available CockroachDB cluster running v22.1 GA or later.
 
+The following databases are supported for use as source databases in AWS DMS:
+
+- Oracle 
+- Microsoft SQL Server
+- Microsoft Azure SQL
+- MySQL-compatible databases (including MariaDB)
+- PostgreSQL-compatible databases (including Amazon Aurora)
+- SAP ASE
```
Comment (stbof) [inline v22.1/aws-dms.md:35]:
```suggestion
- SAP ASE
```

File: v22.1/aws-dms.md [hunk 2]
```diff
@@ -25,6 +25,18 @@ Ensure the following items are completed prior to starting this tutorial:
...
 
+The following databases are supported for use as source databases in AWS DMS:
+
+- Oracle 
+- Microsoft SQL Server
+- Microsoft Azure SQL
+- MySQL-compatible databases (including MariaDB)
+- PostgreSQL-compatible databases (including Amazon Aurora)
+- SAP ASE 
+- MongoDB/Amazon DocumentDB (with MongoDB compatibility)
```
Comment (stbof) [inline v22.1/aws-dms.md:36]:
```suggestion
- MongoDB/Amazon DocumentDB (with MongoDB compatibility)
```

File: v22.1/aws-dms.md [hunk 3]
```diff
@@ -25,6 +25,18 @@ Ensure the following items are completed prior to starting this tutorial:
...
+The following databases are supported for use as source databases in AWS DMS:
+
+- Oracle 
+- Microsoft SQL Server
+- Microsoft Azure SQL
+- MySQL-compatible databases (including MariaDB)
+- PostgreSQL-compatible databases (including Amazon Aurora)
+- SAP ASE 
+- MongoDB/Amazon DocumentDB (with MongoDB compatibility) 
+- Amazon S3
```
Comment (stbof) [inline v22.1/aws-dms.md:37]: S3 is an object store, not a database. Does that matter?

File: v22.1/aws-dms.md [hunk 4]
```diff
@@ -25,6 +25,18 @@ Ensure the following items are completed prior to starting this tutorial:
 - Configure a [replication instance](https://docs.aws.amazon.com/dms/latest/userguide/CHAP_ReplicationInstance.html) in AWS.
 - Ensure you have a secure, publicly available CockroachDB cluster running v22.1 GA or later.
 
+The following databases are supported for use as source databases in AWS DMS:
+
+- Oracle
```
Comment (stbof) [inline v22.1/aws-dms.md:30]:
```suggestion
- Oracle
```

**Review by stbof — CHANGES_REQUESTED** [review]
See space nits and question.

File: v22.1/aws-dms.md (hunk 2 above)
Comment (stbof) [reply v22.1/aws-dms.md:36]: Should MongoDB/Amazon DocumentDB (with MongoDB compatibility) be on two separate lines?

File: v22.1/aws-dms.md (hunk 3 above)
Comment (otan) [reply v22.1/aws-dms.md:37]: i let this and the mongo one go because i was thinking "this is what is supported as source databases in AWS DMS", but i guess i'd say it'd be safe to delete the SAP/Mongo/S3 lines.

**Review by awoods187 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/14526)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @awoods187, @nickvigilante, @otan, and @stbof)

___
*[`v22.1/aws-dms.md` line 36 at r2](https://reviewable.io/reviews/cockroachdb/docs/14526#-N6sEROaJBwlBtTAr0SP-r2-36:-N6wqp5R1gcVz1kzPmNl:b-h0ayjr) ([raw file](https://github.com/cockroachdb/docs/blob/1cd6deba174a7ea56493c3986856b40135be7829/v22.1/aws-dms.md#L36)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

Should MongoDB/Amazon DocumentDB (with MongoDB compatibility) be on two separate lines?
</blockquote></details>

im fine either way on this one https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html
___
*[`v22.1/aws-dms.md` line 37 at r2](https://reviewable.io/reviews/cockroachdb/docs/14526#-N6sEROaJBwlBtTAr0SP-r2-37:-N6wqyT7--TZPeVDKog_:b-jvbcxq) ([raw file](https://github.com/cockroachdb/docs/blob/1cd6deba174a7ea56493c3986856b40135be7829/v22.1/aws-dms.md#L37)):*
<details><summary><i>Previously, otan (Oliver Tan) wrote…</i></summary><blockquote>

i let this and the mongo one go because i was thinking "this is what is supported as source databases in AWS DMS", but i guess i'd say it'd be safe to delete the SAP/Mongo/S3 lines if it was reframed as "AWS DMS supports migrations from these relational databases...").

actually, i'm not sure if mongo -> postgres even works, maybe it errors. let's clean these up and maybe reword it like above.
</blockquote></details>

i borrowed this from https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html which suggests it is a reasonable source

**Comment by nickvigilante** [comment]
@stbof Good call. I'll reorder them prior to merging.
